hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockProvider.java

```diff
@@ -172,27 +176,24 @@ private static Functions.Function3<String, String, TypedProperties, StorageLockC
 
   @VisibleForTesting
   StorageBasedLockProvider(
-      String ownerId,
-      TypedProperties properties,
-      Functions.Function3<String, Long, Supplier<Boolean>, HeartbeatManager> heartbeatManagerLoader,
-      Functions.Function3<String, String, TypedProperties, StorageLockClient> storageLockClientLoader,
-      Logger logger,
-      HoodieLockMetrics hoodieLockMetrics) {
+          String ownerId,
+          TypedProperties properties,
+          Functions.Function3<String, Long, Supplier<Boolean>, HeartbeatManager> heartbeatManagerLoader,
+          Functions.Function3<String, String, TypedProperties, StorageLockClient> storageLockClientLoader,
+          Logger logger,
+          HoodieLockMetrics hoodieLockMetrics) {
```

Comment:
nit: I think the indentation should be 4 spaces.  Could you check your Java checkstyle configuration in IDE?